docs/User Interfaces.md

```diff
@@ -3,7 +3,40 @@
 ## Phoenix Web Interfaces
 
 Phoenix makes an excellent companion to Nerves applications by offering an easy-to-use, powerful framework to create user interfaces in parallel with Nerves device code.
-The easiest way to handle this is to lay out your application as an Umbrella.
+
+### Set up your application
+
+Although it's possible to lay out your application as an Umbrella project, the preferred way is to use poncho projects, i.e. a side-by-side project layout. For the reasoning behind this, please read the [Poncho Projects](http://embedded-elixir.com/post/2017-05-19-poncho-projects/) blog post by Greg Mefford.
```

Comment:
I think we should lower-case both "Umbrella" and "Poncho" everywhere for consistency, since I don't think either is intended to be a proper name. The first sentence is also a little awkward because of the "i.e." in the middle. Maybe something like:

> Although Nerves supports umbrella projects, the preferred project structure is to simply use separate Mix projects side-by-side with `path` dependencies between them if they're in the same source code repository. We call this a "poncho project" structure.

I also don't think you need to credit me as writing the blog post on this page. Maybe something like:

> For the reasoning behind this, please see [this blog post describing poncho projects](http://embedded-elixir.com/post/2017-05-19-poncho-projects/).